Library/Formula/bitlbee.rb

```diff
@@ -88,4 +88,9 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    output = shell_output("#{sbin}/bitlbee -V", 1)
+    assert_match version.to_s, output
```

Comment:
`system` fails the test if the executable returns a non-zero exit code and as Bitlbee returns 1 for the help/version flags it would fail here if we used it.

I could change it to `shell_output("#{sbin}/bitlbee -V", 1)` if you want which would probably be effectively the same.